### ISSUE TITLE
Feature request: Possibility to reverse order of days in rendering

### ISSUE THREAD

**Issue by eskholm** [FamousWolf/week-planner-card#147]
My usecase is a dashboard that shows events that happened in the past week. This works beautifully on a PC monitor,  with 7 days ago on the left, and today on the right side of the monitor.

On mobile though, the view naturally wraps, and the oldest day is the first to be shown and scrolling further down brings us to today.

I would love to be able to reverse the display of days in the configuration. Then I could have one week-planner shown for normal screens and a reversed week-planner shown for mobile devices, and both platforms would feel natural.

So tl;dr:
Please implement "reverse" flag in card config, that does this._days.reverse() in the code before rendering :)

**Comment by FamousWolf** [comment]
You can do this using CSS and card_mod.

```yaml
card_mod:
  style: |
    div.container {
      flex-direction: column-reverse;
    }
```
